tests/Samples.Server.Tests/ResponseTests.cs

```diff
@@ -37,5 +37,20 @@ public async Task Wrong_Query_Should_Return_Error()
             var response = await SendRequestAsync("Oops");
             response.ShouldBe(@"{""errors"":[{""message"":""Body text could not be parsed. Body text should start with '{' for normal graphql query or with '[' for batched query.""}]}");
         }
+
+        [Fact]
+        public async Task Should_Be_Able_To_Deserialize_Variables()
+        {
```

Comment:
Maybe `Should_Be_Able_To_Serialize_Variables` ?

Reply:
The test is to ensure that deserialization from stream to GraphQLRequest works when there are variables in the json. But I guess cuz of the BaseTests class which serializes it too, it does both.

Reply:
Renamed the methods to be less specific. Ideally I'd have queries that actually accept a real variable and then return the value and verify it or something, but need to keep moving and don't want to over-complicate the sample schema.

Reply:
ok